scripts/templates/javascript-project/config/default.yaml

```diff
@@ -66,6 +66,10 @@ server:
     # the process to shut down.
     shutdownGracePeriod: 15
 
+    # Set the maximum HTTP POST (in KB) data request size for
+    # this server
+    maxRequestSize: 512
```

Comment:
I've seen used elsewhere on this project, so sounds reasonable to use it here as well.

Reply:
Actually, we don't (not that I could find). Unless we'd use it everywhere, I am not sure I should be introducing this as part of this PR (could be a good idea as part of a PR dealing specifically with that though).

Reply:
I think it could well be part of this PR, because you're introducing a new feature here. Introducing it later would mean changing the configuration schema, which you can avoid by doing it now.

Reply:
Be that as it may, this is not how bytes are configured by standard in MAGE. I don't want to have people try to remember which config entry takes which kind of numbers.

Reply:
Do we have that many other cases where we specify size in bytes in config?

Reply:
Not sure, I would have to scan the repository to find such cases - which is why I would argue it should be a task to be done separately.